### ISSUE TITLE
New feature idea: Contributors page to include SiG/CoP

### ISSUE THREAD

**Issue by nomadscientist** [galaxyproject/training-material#4420]
It could be cool if people are in a Community of Practice if this was automatically shown on their Contributor page.

**Comment by hexylena** [comment]
yeah so with #4408 we're considering adding an "affiliation" field where people could affiliate themselves with specific organisations, it would definitely be possible to have an organisation that was a CoP and then show all the user's orgs on their page.

**Comment by nomadscientist** [comment]
Nice!
